packages/apps/frenet/src/frenet-events.ts

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable import/prefer-default-export */
+import '@cloudcommerce/firebase/lib/init';
+import {
+  createAppEventsFunction,
+  ApiEventHandler,
+} from '@cloudcommerce/firebase/lib/helpers/pubsub';
+import config from '@cloudcommerce/firebase/lib/config';
+import functions from 'firebase-functions/v1';
+import handleTrackingCodes from './functions-lib/tracking-codes';
+import removeDeliveredToFirestore from './functions-lib/remove-delivered';
+import db from './functions-lib/database';
+import handleApiEvent from './functions-lib/events-to-frenet';
+
+const { httpsFunctionOptions: { region } } = config.get();
+
+export const frenet = {
+  cronTrackingCodes: functions.region(region).pubsub
+    .schedule('*/30 * * * *')
+    .onRun(() => {
+      return handleTrackingCodes;
+    }),
+
+  cronRemoveDelivered: functions.region(region).pubsub
+    .schedule('*/30 * * * *')
+    .onRun(() => {
+      return removeDeliveredToFirestore;
+    }),
+
+  cronRemoveOldTrackingCodes: functions.region(region).pubsub
+    .schedule('30 */24 * * *')
+    .onRun(() => {
+      return db.clear();
+    }),
```

Comment:
Isso aqui tá muito legal não 😬 
Pensa só, toda loja, mesmo não usando Frenet, nessa implementação vai executar 2 funções a cada meia hora e mais uma diariamente, isso não é muito legal não.
O ideal é não ter nenhuma scheduled function em app justamente pra não ficar tendo chamada desnecessária, 3 então piorou haha
Se realmente for necessário, use uma scheduled só, você chama de 30 em 30 minutos (como já está fazendo aqui) e dentro da função você verifica quais métodos precisa executar, pelo horário ou por algum estado salvo no banco (claro que só vai fazer isso se o app estiver instalado), faz sentido?

Reply:
Além das chamadas desnecessárias, fazer o deploy de 3 funções é pior do que de uma só. Uma função não precisa (em geral nem deve) executar só um método, não vamos levar microserviços tão a ferro e fogo (fica caro e mais difícil dar manutenção 🙃 )